localization/english/wc_traits_l_english.yml

```diff
@@ -0,0 +1,1027 @@
+ trait_bunny_character_desc:0 "$trait_bunny_desc$"
+ trait_creature_hozen:0 "Hozen"
+ trait_creature_hozen_desc:0 "The hozen are a race of small humanoids that resemble ordinary monkeys, but unlike their other primate cousins, the average intelligence of the hozen is slightly higher than an average monkey's to the point of being able to communicate verbally between each other and sometimes other races as well. Nevertheless they are still very primitive, and more often than not, mischievous to the point of causing trouble for more civilised races, although there are exceptions to the rule."
+ trait_creature_hozen_character_desc:0 "This character is a hozen. $trait_creature_hozen_desc$"
+ trait_creature_pygmy:0 "Pygmy"
+ trait_creature_pygmy_desc:0 "This character is a pygmy."
+ trait_creature_pygmy_character_desc:0 "$trait_creature_pygmy_desc$"
+ trait_creature_goblin:0 "Goblin"
+ trait_creature_goblin_desc:0 "This character is a goblin."
+ trait_creature_goblin_character_desc:0 "$trait_creature_goblin_desc$"
+ trait_aspect_of_death:0 "Aspect of Death"
+ trait_aspect_of_death_desc:0 "The Aspect of Death was once known as the Aspect of Earth, who has been chosen by Khaz'goroth and given dominion over the earth and the deep places of the world, but darkness swallowed aspect's soul."
+ trait_aspect_of_death_character_desc:0 "$trait_aspect_of_death_desc$"
+ trait_aspect_of_life:0 "Aspect of Life"
+ trait_aspect_of_life_desc:0 ""This is my gift: compassion for all living things. A drive to protect and nurture them. And the ability to heal that which others cannot, birth what others may not, and love even the unlovable � who surely need such grace more than any other souls." \n- Eonar the Life-Binder"
+ trait_aspect_of_life_character_desc:0 "$trait_aspect_of_life_desc$"
+ trait_being_undead:0 "Undead"
+ trait_being_undead_desc:0 "The literal meaning of undead is simple - a creature or person risen from death. The undoing of death, hence undead. While they vary in size, shape, and form, the most common undead are those who have risen from their own (or other's) accord, who maintain their sanity and memories from life. This character in question is a textbook undead, and may act differently from every other undead creature due to their own autonomy."
+ trait_being_undead_character_desc:0 "$trait_being_undead_desc$"
+ trait_creature_ghoul:0 "Ghoul"
+ trait_creature_ghoul_desc:0 "Men who have finally been given the grace of undeath are known as ghouls. Varying in size, shape, and form, ghouls are the shocktroopers of every undead mass. Using speed as their advantage in battle, these charging hulks of fallen warriors are every shieldbearer's worst nightmare."
+ trait_creature_ghoul_character_desc:0 "$trait_creature_ghoul_desc$"
+ trait_creature_wraith:0 "Spirit"
+ trait_creature_wraith_desc:0 "This character is an incorporeal spirt."
+ trait_creature_wraith_character_desc:0 "$trait_creature_wraith_desc$"
+ trait_lich_king:0 "Lich [Root.GetQueenKing]"
```

Comment:
Does the scope still work?

Reply:
> Does the scope still work?

Definitely not, I'll adjust it once the trait is here. So far, only races.